client/components/relatedWidget/ProductList.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ProductListItem from './ProductListItem.jsx';
+
+const ProductList = (list, selected, callback) => {
```

Comment:
If you are doing object destructuring here shouldn't all the arguments be wrapped in `{}`
so the line would be: 
`const ProductList = ({list, selected, callback}) => {`